gnocchi/tests/test_carbonara.py

```diff
@@ -256,10 +256,10 @@ def test_74_percentile_serialized(self):
              (datetime64(2014, 1, 1, 12, 0, 4), 5),
              (datetime64(2014, 1, 1, 12, 0, 9), 6)])
         ts = self._resample(ts, numpy.timedelta64(60, 's'), '74pct')
-        ts.merge(saved_ts)
+        saved_ts.merge(ts)
```

Comment:
i don't like how we swapped positions here but i'm not completely worried about this.